doc/helpers/Input-extensions.md

```diff
@@ -8,11 +8,12 @@ Provides various attached properties for _input controls_, such as `TextBox` and
 
```

Comment:
@rajamatt It should be `soft keyboard` instead of `soft-keyboard`.
Only in uno.toolkit.ui we are using "soft-keyboard" at the moment, if you can please do the update everywhere.
https://github.com/search?q=org%3Aunoplatform%20%22soft-keyboard%22&type=code
Thanks in advance.